lib/cli/test/unit/Cardano/CLISpec.hs

```diff
@@ -243,6 +243,8 @@ spec = do
             , "  list                     List all known wallets."
             , "  create                   Create a new wallet using a sequential"
             , "                           address scheme."
+            , "  restore                  Restore a wallet using a public"
```

Comment:
`create` and `restore` are actually two synonyms for the wallet :) 

What about creating a sub-command of `create` in the form of 

`create from-mnemonic`  and `create from-public-key`

Reply:
done in https://github.com/input-output-hk/cardano-wallet/pull/1421/commits/a2b4df64a33184d951009ff9c34e600da8d6fc3a and https://github.com/input-output-hk/cardano-wallet/pull/1421/commits/4cf7ba4b9df864e1e7d41504f653e1054a83f440